src/transformers/modeling_outputs.py

```diff
@@ -65,6 +65,23 @@ class BaseModelOutputWithNoAttention(ModelOutput):
     hidden_states: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class BaseModelOutputWithIntermediateActivations(ModelOutput):
```

Comment:
Not sure this output class has any use case besides DPT, better to define the custom model output class in modeling_dpt.py

Reply:
I personally prefer to put it outside DPT modeling file for consistency with other dataclasses, otherwise I am afraid it will add too much code on the dpt modeling file

Reply:
As you have also done the same process for Tapas, DETR, etc. Let's move them

Reply:
Moved the dataclasses in 275f25c